cmake/checks/check_01_cxx_features.cmake

```diff
@@ -463,3 +468,64 @@ CHECK_CXX_SOURCE_COMPILES(
   "
   DEAL_II_HAVE_ISFINITE)
 
+
+#
+# Check that we can use feenableexcept through the C++11 header file cfenv:
```

Comment:
Oh, this is exactly the point! :-D

The previous situation tested for ```<fenv.h>``` and used ```<cfenv>```, and this happens [1].
The most sensible strategy is now to stick with ```<cfenv>```, because (a) there might be a
reasons why it is used (that I might not be aware of ;-) ), (b) it is offered by the language standard.



[1] http://cdash.kyomu.43-1.org/buildSummary.php?buildid=11733